src/main/app/src/app/zaken/zaak-create/zaak-create.component.ts

```diff
@@ -139,9 +130,9 @@ export class ZaakCreateComponent implements OnInit {
         this.actionsSidenav.close();
     }
 
-    locatieGeselecteerd(locatie: { naam: string, coordinates: GeometryCoordinate }): void {
+    locatieGeselecteerd(locatie: any): void {
```

Comment:
Zo te zien heeft locatie een weergavenaam. Valt er voor de rest niets meer te zeggen over locatie dan 'any'? (kan best zijn dat dat zo is maar ik vraag het maar)